src/app/customer-management/customer-management.component.html

```diff
@@ -0,0 +1,14 @@
+<div id="container">
+    <h3 id="page-title">Customer Management</h3>
+    <section id="customers">
+      <button class="btn btn-primary" id="button-add-customer" type="button"><i class="bi bi-plus"></i>Add Customer</button>
+      <table class="table">
+        <tbody>
+          <tr>
+            <td>John James</td>
+            <td><button id="button-view-customer" type="button" class="btn btn-secondary float-end">View</button></td>
```

Comment:
button-view-customer in css, isn't needed.
You just need btn-light class from bootstrap if i'm not mistaken

Reply:
fixed!